DSL.Ruuter/POST/train.yml

```diff
@@ -0,0 +1,8 @@
+train_model:
+  call: http.post
+  args:
+    url: http://ml-service:8000/task
```

Comment:
What endpoint is `http://ml-service:8000/task`?

Reply:
This is the endpoint which is hosted on our ML Service responsible for training a model. It initiates the training of a model based on the corpora.